packages/omgidl-parser/src/parseIDL.test.ts

```diff
@@ -30,7 +30,7 @@ describe("omgidl parser tests", () => {
     struct A {
       int32 num;
     };
-    
+
```

Comment:
If there is no need for changes, please remove those spaces

Reply:
All the suggested changes applied.